boards/grand_central_m4/Cargo.toml

```diff
@@ -12,32 +10,34 @@ edition = "2021"
 repository = "https://github.com/atsamd-rs/atsamd"
 readme = "README.md"
 
-[dependencies]
-ws2812-timer-delay = "0.3"
-
 [dependencies.cortex-m-rt]
 version = "0.7"
 optional = true
 
 [dependencies.atsamd-hal]
-version = "0.15"
+path = "../../hal"
+version = "0.17.0"
```

Comment:
The `grand_central_m4` is a Tier 2 BSP; it should depend on the `crates.io` version of the HAL, and not specify a path dependency.

Reply:
ahh ok